test/fixtures/misc-templates/deployment-w-progress.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: extensions/v1beta1
```

Comment:
So I tried this and was able to add the `progressDeadlineSeconds` into the spec but adding a `readinessProbe` is not as easy. We need a `readinessProbe` so that the container fails the ready check causing the timeout